### ISSUE TITLE
About the config file

### ISSUE THREAD

**Issue by miaohx666** [akanazawa/human_dynamics#12]
First of all, I think the model is very cool. So I really want to run the offered model by myself.
But when I run the command : "python -m demo_video --vid_path demo_data/penn_action-2278.mp4", there is a erroe :''[!] You need to specify `load_path` to load a pretrained model''.
I think it may need a config file. So would you mind offer the config file to me? Thanks very much.

**Comment by jasonyzhang** [comment]
Pass the pretrained model path using `--load_path models/hmmr_model.ckpt-2699068`

**Comment by miaohx666** [comment]
Thanks for answering. And would you tell me  what to input next ?
![VUST}~` {I_Y$N7NLCWHXBL](https://user-images.githubusercontent.com/51950700/61350865-8d910700-a89b-11e9-8726-ace639a84545.png)

**Comment by jasonyzhang** [comment]
Make sure that you've downloaded the pretrained models and then point the path to one of the checkpoints in the models folder

**Comment by miaohx666** [comment]
I am so glad to get your answer.
I have found a small problem which I want to tell you.
In the website 'http://angjookanazawa.com/cachedir/hmmr/hmmr_models.tar.gz', I installed the flies below.
![QQ图片20190723170150](https://user-images.githubusercontent.com/51950700/61698657-08f72a80-ad6c-11e9-9c9d-1ae3795fb9f6.png)

There are not 'hmmr_model.ckpt-2699068' which you write in readme.md:
'# Run on a single video:
python -m demo_video --vid_path demo_data/penn_action-2278.mp4 --load_path models/hmmr_model.ckpt-2699068'

When I run demo_video.py , the error is 'NotFoundError (see above for traceback): Key resnet_v2_50/block1/unit_1/bottleneck_v2/conv1/BatchNorm/beta not found in checkpoint'

Thanks for reading at last.

**Comment by jasonyzhang** [comment]
Should be ckpt 1119816 then. I'll update the README
